### ISSUE TITLE
number blocked

### ISSUE THREAD

**Issue by kunalsmehtajobs** [mgp25/Chat-API-NET#139]
Hi,
i used iPhone settings to bypass the encryption and got my number blocked.
@mgp25 When do we get the encryption implemented in our api?
Any help is highly appreciated.

Regards
Kunal

**Comment by cfristofer** [comment]
Exists any other place, when we can follow the updates of this api. Thanks

**Comment by mym2o** [comment]
I have an iPhone and got my number blocked too. Furthermore, I receive some advertisements from an unknown number (the same which sent me the pwd when I used WART).
The first time, I was unbanned after a day, but some days ago I was banned again. But this time I was using only WhatsApp Messenger official iPhone app!!! Is mgp25 stolen my number to do any kind of spam activity?

Regards
Domenico

**Comment by mgp25** [comment]
@mym2o wtf i just read?

1) WART only requests an sms code which WhatsApp send to you
2) You can't use your account in the API and in your phone at the same time
3) Do you even read the code?
4) Not possible

**Comment by mym2o** [comment]
I know that I can't use my account both on my phone and in the API. It will be like to use the same account on two smartphones. In fact, I have uninstalled first and then I used it on your example "chat code". I immediately received an sms code from an unusual number (+34 or something like this, but I always received whatsapp sms from prefix +49) and then I was banned. I'm only waiting for WhatsApp answer to my request to be sure.
